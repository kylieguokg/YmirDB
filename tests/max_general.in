set a 1000
list entries
set b a 20 3999
list entries
max a
max b
set c 2000
set d 99999999
list entries
append c b
list entries
max a
max b
max c
max d
set d 1000
set c 1 d
set b 2 c
set a b 999
list entries
max a
max b
max c
max d
bye
