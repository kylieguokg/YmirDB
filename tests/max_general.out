> ok

> a [1000]

> ok

> b [a 20 3999]
a [1000]

> 1000

> 3999

> ok

> ok

> d [99999999]
c [2000]
b [a 20 3999]
a [1000]

> ok

> d [99999999]
c [2000 b]
b [a 20 3999]
a [1000]

> 1000

> 3999

> 3999

> 99999999

> ok

> ok

> ok

> ok

> d [1000]
c [1 d]
b [2 c]
a [b 999]

> 1000

> 1000

> 1000

> 1000

> bye
